scripts/package

```diff
@@ -11,14 +11,26 @@ TAG=${TAG:-${VERSION}}
 REPO=${REPO:-longhornio}
 IMAGE=${REPO}/${PROJECT}:${TAG}
 
+case $(uname -m) in
+	aarch64 | arm64)
+		ARCH=arm64
+		;;
+	x86_64)
+		ARCH=amd64
+		;;
+	*)
+		echo "$(uname -a): unsupported architecture"
+		exit 1
+esac
+
 if [ ! -x ./bin/longhorn ]; then
     ./scripts/build
 fi
```

Comment:
Just asking for clarification, it looks like we are always expecting to be building on the distribution os (amd build on an amd, arm build on an arm instance) ?

Reply:
Yes, I think currently this is the way.